src/test/java/unit/FineGrainedAccessGcsObjectReference.java

```diff
@@ -185,7 +183,7 @@ void updateObjectReferenceWithBucketLevelAccess() throws IOException {
 
   @Test
   @DisplayName("user tries to update reference of a bucket object that they don't have access to")
-  void attemptToUpdatePrivateBucketObject() throws IOException {
+  void updatePrivateBucketObject() throws IOException {
```

Comment:
Should this one still be `attempt...`? It looks like it's expected to fail since workspace READERS can't modify referenced resources